source/engine/Sekai.Engine/Entity.cs

```diff
@@ -29,6 +29,11 @@ public IEnumerable<Entity> Entities
         }
     }
 
+    /// <summary>
+    /// The transform of the entity
+    /// </summary>
+    public Transform? Transform { get; set; }
```

Comment:
The entity should never have any references to any components. They should only be obtained via `GetComponent<T>()` or similar methods.

Reply:
How do we plan to apply transform on a entity if we don't keep a reference in the object?

Reply:
there will be entity processors responsible for accessing components of a given type